### ISSUE TITLE
LBRY network status indicator in App

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-app#603]
## The Issue
As a LBRY user, I want to make sure that my PC and network configuration allows for the sharing and consuming content.  We should have some type of indicator which checks connectivity on the daemon TCP port (3333) and the DHT UDT port (4444). 

For content sharing to properly work, hosts need port 3333 accessible and open to the internet. If the port is not forwarded correctly, their PC won't be able to host but it may still be able to reach out to the reflector servers.  If something is blocking port 3333, it won't be able to reach out to the reflector servers, so the content won't be available at all. 

For content consumption to work, LBRY needs to be able to connect to the DHT network to find peers.  If something is blocking the port, the user experience is really poor - all downloads/streams will hang. There is no indication that something is wrong or needs to be changed.  

Background: I've run into a couple users whose ISP blocks port 4444 and they were not able to stream until the port was changed to something else.  We also have users who run mining software on port 3333, so if this is taken up by the network, they won't have a proper daemon connection. 

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by btzr-io** [comment]
this? -> https://lbryio.github.io/lbry/cli/#status

**Comment by MSFTserver** [comment]
for the last paragraph i think we should add fallback ports if it detects the ports are in use by mining or other causes use a fallback

**Comment by tzarebczan** [comment]
@btzr-io nope, not as straightforward as that. I'm not exactly sure what the best technical solution to determine if the connections on the daemon port and dht port are working proprly

**Comment by kauffj** [comment]
I think this is a ticket for `lbry`, not `lbry-app`. It's the job of the daemon to assess network connectivity. The daemon can be modified to provide more feedback about network status via the `status` call.

**Comment by kauffj** [comment]
Or rather, it's a ticket for both, and this ticket is blocked until `lbry` has a modified `status` call that can report network connectivity.

**Comment by tzarebczan** [comment]
Opened https://github.com/lbryio/lbry/issues/934 on the daemon side.
